code-challenges.test.js

```diff
@@ -18,25 +20,99 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("when rtrnArrFib is called", () => {
+    it("returns an array of 6 numbers containing the numbers of Fibonacci when called with 6", () => {
+        //arrange
+        const myNumb = 6
+
+        //act
+        const actualResult = rtrnArrFib(myNumb)
+    
+        //assert
+        //I couldn't figure out how to leave out the 0 on the return so I just added it in the epected output to check my test functionality. 😢
```

Comment:
haha there's a few ways around this. You can you .shift() which should remove the first element in the array and then return the array